test/Mvc.Analyzers.Test/TestFiles/AddResponseTypeAttributeCodeFixProviderIntegrationTest/CodeFixWithConventionMethodAddsMissingStatusCodes.Output.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Microsoft.AspNetCore.Mvc.Analyzers._OUTPUT_
+{
+    [ApiController]
+    [Route("[controller]/[action]")]
+    public class CodeFixWithConventionMethodAddsMissingStatusCodes : ControllerBase
+    {
+        [ApiConventionMethod(typeof(DefaultApiConventions), nameof(DefaultApiConventions.Find))]
```

Comment:
@rynowak thoughts on leaving this around?

Reply:
It should remove this, it's not meaningful